src/e2e-test/java/io/cdap/plugin/zendesk/actions/ZendeskPropertiesPageActions.java

```diff
@@ -149,13 +143,17 @@ static boolean compareValueOfBothResponses(String zendeskResponse, String bigQue
     Map<String, Object> bigQueryResponseInMap = gson.fromJson(bigQueryResponse, type);
     MapDifference<String, Object> mapDifference = Maps.difference(zendeskResponseInmap, bigQueryResponseInMap);
     logger.info("Assertion :" + mapDifference);
-
     return mapDifference.areEqual();
   }
 
   public static void getBigQueryTableData(String dataset, String table, BigInteger uniqueId)
```

Comment:
We have written the altered query here specifically for groups but this method ``` getBigQueryTableData  ``` is common and can be used to fetch data from any big query table, this will throw an error if the table consist data of another object. (For ex: Tags may not have field like updatedAt and createdAt).

Reply:
The objects we are using : Groups and Organisations , both have the same fields.